packages/block-library/src/media-text/style.scss

```diff
@@ -1,12 +1,10 @@
 .wp-block-media-text {
 	display: grid;
-}
```

Comment:
It appears this one was resolved separately in #13989, causing the merge conflict. I'll plan to resolve it.